src/robot.js

```diff
@@ -516,8 +516,10 @@ class Robot {
   // Returns nothing.
   parseHelp (path) {
     const scriptDocumentation = {}
-    const body = fs.readFileSync(path, 'utf-8')
-    const lines = body.split('\n')
+    const body = fs.readFileSync(require.resolve(path), 'utf-8')
+
+    const useStrictHeaderRegex = /^["']use strict['"];?\s+/
```

Comment:
the `'use strict'` statement on top of the files is needed for Node@4 compatibility